app/sections/about/about.component.html

```diff
@@ -18,95 +18,6 @@
-                                                                                    value="Donation"><input
-            type="hidden" name="sc-amount" class="sc_amount" value=""><input type="hidden" name="sc-redirect"
-                                                                             value="/donations-thank-you/"><input
-            type="hidden" name="sc-redirect-fail" value="http://www.gapminder.org/about-gapminder/history/"><input
-            type="hidden" name="sc-currency" value="USD"><input type="hidden" name="stripeToken" value=""
-                                                                class="sc_stripeToken"><input type="hidden"
-                                                                                              name="stripeEmail"
-                                                                                              value=""
-                                                                                              class="sc_stripeEmail"><input
-            type="hidden" name="wp-simple-pay" value="1"><input type="hidden" name="sc-details-placement" value="above"><input
-            type="hidden" id="wp-simple-pay-pro-nonce" name="wp-simple-pay-pro-nonce" value="816663c166"><input
-            type="hidden" name="sc_sub_id" class="sc_sub_id" value=""><input type="hidden" name="sc_sub_interval"
-                                                                             class="sc_sub_interval"
-                                                                             value="month"><input type="hidden"
-                                                                                                  name="sc_sub_interval_count"
-                                                                                                  class="sc_sub_interval_count"
-                                                                                                  value="1"><input
-            type="hidden" name="sc_sub_statement_description" class="sc_sub_statement_description" value="">
-            <button class="sc-payment-btn stripe-button-el"><span>Donate now</span></button>
-          </form>
-        </small>
-
-        <p>Swish number: 123 327 55 75</p>
-      </div>
-    </div>
-  </div>
-  <div id="text-18" class="widget widget_text widget-widget_text">
-    <div class="widget-wrap widget-inside">
-      <div class="textwidget"><h5>Post Address: </h5>
-        <p>Stiftelsen Gapminder<br>
-          Box 38025<br>
-          100 64 Stockholm</p>
-        <h5>Billing Address: </h5>
-        <p>Stiftelsen Gapminder<br>
-          FE 586<br>
-          Kund ID PMA5334<br>
-          105 69 Stockholm</p>
-        <h5>Organisation number</h5>
-        <p>802424-7721</p>
-      </div>
-    </div>
-  </div>
-
-  <div style="clear: both"></div>
-</div>
-
+<div class="col-md-4">
+  <sidebar></sidebar>
+</div>
```

Comment:
new line ;) probably you are able to change the settings in your editor to add it automatically on save